### PR TITLE
Debut Anniversary Event

### PR DESCRIPTION
**Pull request by Arieswaran**
need to set up the event config

### CODE REVIEW

File: cogs/anniversary.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    async def assignAllCardsToBot(self, ctx: commands.Context) -> None:
+        """Assigns all cards to the bot"""
+        if ctx.author.id in func.settings.ADMIN_IDS:
+            #get all anniversary sale cards and give it to the bot
+            all_cards = iufi.GetAllCards()
+            for card_id, card_price in all_cards:
+                card = iufi.CardPool.get_card(card_id)
+                if card.owner_id:
+                    continue
+                await func.update_card(card_id, {"$set": {"owner_id": self.bot.user.id}})
```
Comment (ChocoMeow) [inline cogs/anniversary.py:38]: We can update all the cards with single request
```py
await func.update_card(card_ids, {"$set": {"owner_id": self.bot.user.id}})
```

File: cogs/anniversary.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    async def removeAssignedCards(self, ctx: commands.Context) -> None:
+        """Removes all assigned cards from the bot"""
+        if ctx.author.id in func.settings.ADMIN_IDS:
+            all_cards = iufi.GetAllCards()
+            for card_id, card_price in all_cards:
+                card = iufi.CardPool.get_card(card_id)
+                if card.owner_id != self.bot.user.id:
+                    continue
+                await func.update_card(card_id, {"$set": {"owner_id": None}})
+            await ctx.reply("All assigned cards have been removed.")
```
Comment (ChocoMeow) [inline cogs/anniversary.py:51]: We can update all the cards with single request >.<
```py
await func.update_card(card_ids, {"$set": {"owner_id": self.bot.user.id}})
```

File: settings.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "MAIN_GUILD": 781828787352240129,
     "MUSIC_TEXT_CHANNEL": 1216685839585771560,
     "MUSIC_VOICE_CHANNEL": 1216674440486588457,
-    "ALLOWED_CATEGORY_IDS": [987352501172989993, 1144810748158165043],
+    "ALLOWED_CATEGORY_IDS": [987352501172989993, 1144810748158165043, 1143358482494533662],
```
Comment (ChocoMeow) [inline settings.json:7]: which channel this ID is linked to? >.<

**Review by ChocoMeow — CHANGES_REQUESTED** [review]
Wow, that’s an amazing number of updates! Thank you so much for all your hard work! I really appreciate it! <3

File: settings.json (hunk 3 above)
Comment (Arieswaran) [reply settings.json:7]: It's my test server category. Added it for testing in that. will remove it

**Review by ChocoMeow — APPROVED** [review]

**Pull request closed without merging**